### DISCUSSION TITLE
GLPI Agent Android Version

### DISCUSSION THREAD

**Discussion by ErrafayM** [glpi-project/glpi-agent#765] [Q&A]
### Bug reporting acknowledgment

Yes, I read it

### Professional support

None

### Describe the bug

There is an issue with the latest android version of glpi agent , i have tested in a tablette and a phone and both have same problem when i download the package via Play Store, it cannot run. 

### To reproduce

1. Download Android GLPI Agent via PlayStore
2. Run the GLPI Agent
3. Application Crashes.

### Expected behavior

It should run correctly.

### Operating system

Other (See additional context below)

### GLPI Agent version

v1.5

### GLPI version

10.0.16

### GLPIInventory plugin or other plugin version

GLPI Inventory v1.3.5

**Comment by g-bougard** [comment]
Hi @ErrafayM 

[android glpi-agent project](https://github.com/glpi-project/android-inventory-agent) is not directly related to this glpi-agent project which is the perl agent one. So I moved your issue as a discussion.

@stonebuzz have you an answer for @ErrafayM ?

**Comment by stonebuzz** [comment]
Hi @ErrafayM 

Can you confirm that the second time the application is launched, it starts up correctly? 

This is the result I get on my phone .

**Comment by stonebuzz** [comment]
Hi @ErrafayM after a few tests,

this only happens if you launch app immediately after installation

if you wait 3 seconds, the application launches correctly.

Can you confirm this?

I think the _homemade_ splashscreen is the cause.

Android forces you to migrate to its `SplashScreen`  tool and does not appear to be stable in terms of compatibility with the old method

**Reply by ErrafayM** [reply]
Thank you for the answer. But sadly it does not work on my phone and neither on the tablets tried on many but only one did work and it's running Android 13. 
My phone and the other tablets are running with Android 14. 
![Screenshot_20240918_204222_GLPI Agent](https://github.com/user-attachments/assets/650dca3e-3989-4208-97f3-938bc1842ce5)

**Reply by ErrafayM** [reply]
I also tried to uninstall and install and wait few seconds as you suggested but also not working
